internal/buildscripts/packaging/installer/install.sh

```diff
@@ -1196,17 +1195,33 @@ parse_args_and_install() {
       --without-systemd-instrumentation)
         with_systemd_instrumentation="false"
         ;;
-      --with-java-instrumentation)
-        with_java_instrumentation="true"
-        ;;
-      --without-java-instrumentation)
+      --with-instrumentation-sdk)
         with_java_instrumentation="false"
-        ;;
-      --with-node-instrumentation)
-        with_node_instrumentation="true"
-        ;;
-      --without-node-instrumentation)
         with_node_instrumentation="false"
+        echo "$2" | tr ',' '\n' | while read lang; do
+            if [[ "$lang" -eq "java" ]]; then
+                with_java_instrumentation="true"
+            elif [[ "$lang" -eq "node" ]]; then
+                with_node_instrumentation="true"
+            else
+                echo "Unknown instrumentation SDK $1" >&2
+                usage
```

Comment:
```suggestion
                usage
                echo "[ERROR] Unknown instrumentation SDK $1" >&2
```
Since the usage message is pretty long and dense, the error may not be noticeable.